Update reqwest to 0.12

The current proxy v2 branch (https://github.com/freedomofpress/securedrop-client/pull/1718) has reqwest 0.11, we should update to 0.12 before releasing it. I don't want to do it in that branch since it's already so big.

When I upgrade to reqwest 0.12:
```
============================= test session starts ==============================
platform linux -- Python 3.11.9, pytest-7.4.4, pluggy-1.4.0
rootdir: /home/user/github/freedomofpress/securedrop-client/proxy
plugins: httpbin-2.0.0
collected 16 items                                                             

tests/test_errors.py .....F                                              [ 37%]
tests/test_proxy.py .....F....                                           [100%]

=================================== FAILURES ===================================
_____________________________ test_cannot_connect ______________________________

proxy_request = <function proxy_request.<locals>.proxy_ at 0x7a8b6f488720>

    def test_cannot_connect(proxy_request):
        test_input = {
            "method": "GET",
            "path_query": "/",
            "stream": False,
        }
        # .test is a reserved TLD, so it should never resolve
        result = proxy_request(input=test_input, origin="http://missing.test/foo")
        assert result.returncode == 1
>       assert (
            result.stderr.decode().strip()
            == '{"error":"error sending request for url (http://missing.test/): '
            + "error trying to connect: dns error: failed to lookup address information: "
            + 'Name or service not known"}'
        )
E       assert '{"error":"er...sing.test/)"}' == '{"error":"er...e not known"}'
E         Skipping 52 identical leading characters in diff, use -v to show
E         - ing.test/): error trying to connect: dns error: failed to lookup address information: Name or service not known"}
E         + ing.test/)"}

tests/test_errors.py:67: AssertionError
_________________________________ test_timeout _________________________________

proxy_request = <function proxy_request.<locals>.proxy_ at 0x7a8b6f489760>
httpbin = <pytest_httpbin.serve.Server object at 0x7a8b6f45b590>

    def test_timeout(proxy_request, httpbin):
        start = time.time()
        test_input = {"method": "GET", "path_query": "/delay/10", "stream": False, "timeout": 1}
        result = proxy_request(input=test_input)
        assert result.returncode == 1
>       assert (
            result.stderr.decode().strip()
            == '{"error":"error sending request for url (http://127.0.0.1:%s/delay/10): ' % httpbin.port
            + 'operation timed out"}'
        )
E       assert '{"error":"er...5/delay/10)"}' == '{"error":"er...n timed out"}'
E         Skipping 63 identical leading characters in diff, use -v to show
E         - /delay/10): operation timed out"}
E         + /delay/10)"}

tests/test_proxy.py:54: AssertionError
=========================== short test summary info ============================
FAILED tests/test_errors.py::test_cannot_connect - assert '{"error":"er...sing.test/)"}' == '{"error":"er...e not known"}'
FAILED tests/test_proxy.py::test_timeout - assert '{"error":"er...5/delay/10)"}' == '{"error":"er...n timed out"}'
======================== 2 failed, 14 passed in 17.60s =========================
```

This is fallout from https://github.com/seanmonstar/reqwest/pull/2199 - it's unclear why the change was made, but we can probably keep the old behavior without too much difficulty:
```diff
diff --git a/proxy/src/main.rs b/proxy/src/main.rs
index d1cb6563..b6e04772 100644
--- a/proxy/src/main.rs
+++ b/proxy/src/main.rs
@@ -165,10 +165,12 @@ async fn main() -> ExitCode {
     match proxy().await {
         Ok(()) => ExitCode::SUCCESS,
         Err(err) => {
+            let mut error = err.to_string();
+            if let Some(source) = err.source() {
+                error = format!("{}: {}", error, source);
+            }
             // Try to serialize into our error format
-            let resp = ErrorResponse {
-                error: err.to_string(),
-            };
+            let resp = ErrorResponse { error };
             match serde_json::to_string(&resp) {
                 Ok(json) => {
                     // Print the error to stderr
```

--- a/proxy/Cargo.toml
+++ b/proxy/Cargo.toml
@@ -11,7 +11,7 @@ qubesdb = []
 [dependencies]
 anyhow = {version = "1.0.75"}
 futures-util = "0.3.30"
-reqwest = {version = "0.11.20", features = ["gzip", "stream"]}
+reqwest = { version = "0.12", features = ["gzip", "stream"] }
 serde = {version = "1.0.188", features = ["derive"]}
 serde_json = "1.0.107"
 tokio = {version = "1.0", features = ["macros", "rt"]}

--- a/proxy/src/main.rs
+++ b/proxy/src/main.rs
@@ -165,10 +165,12 @@ async fn main() -> ExitCode {
     match proxy().await {
         Ok(()) => ExitCode::SUCCESS,
         Err(err) => {
+            let mut error = err.to_string();
+            if let Some(source) = err.source() {
+                error = format!("{}: {}", error, source);
+            }
             // Try to serialize into our error format
-            let resp = ErrorResponse {
-                error: err.to_string(),
-            };
+            let resp = ErrorResponse { error };
             match serde_json::to_string(&resp) {
                 Ok(json) => {
                     // Print the error to stderr

--- a/proxy/tests/test_errors.py
+++ b/proxy/tests/test_errors.py
@@ -67,6 +67,5 @@ def test_cannot_connect(proxy_request):
     assert (
         result.stderr.decode().strip()
         == '{"error":"error sending request for url (http://missing.test/): '
-        + "error trying to connect: dns error: failed to lookup address information: "
-        + 'Name or service not known"}'
+        + 'client error (Connect)"}'
     )

--- a/supply-chain/audits.toml
+++ b/supply-chain/audits.toml
@@ -93,6 +93,13 @@ criteria = "safe-to-run"
 delta = "4.0.2 -> 4.3.0"
 notes = "Windows-specific code was not reviewed."
 
+[[trusted.atomic-waker]]
+criteria = "safe-to-deploy"
+user-id = 33035 # Taiki Endo (taiki-e)
+start = "2023-01-15"
+end = "2024-11-17"
+notes = "Rust Project member"
+
 [[trusted.flate2]]
 criteria = "safe-to-deploy"
 user-id = 4333
@@ -128,6 +135,13 @@ start = "2020-10-05"
 end = "2024-08-29"
 notes = "Rust Project member"
 
+[[trusted.h2]]
+criteria = "safe-to-deploy"
+user-id = 359 # Sean McArthur (seanmonstar)
+start = "2019-03-13"
+end = "2024-11-17"
+notes = "see https://github.com/freedomofpress/securedrop-engineering/pull/87"
+
 [[trusted.home]]
 criteria = "safe-to-deploy"
 user-id = 6202 # Eric Huss (ehuss)
@@ -135,12 +149,40 @@ start = "2023-04-25"
 end = "2024-09-12"
 notes = "Rust Project member"
 
+[[trusted.http]]
+criteria = "safe-to-deploy"
+user-id = 359 # Sean McArthur (seanmonstar)
+start = "2019-04-05"
+end = "2024-11-17"
+notes = "see https://github.com/freedomofpress/securedrop-engineering/pull/87"
+
+[[trusted.http-body-util]]
+criteria = "safe-to-deploy"
+user-id = 359 # Sean McArthur (seanmonstar)
+start = "2022-10-25"
+end = "2024-11-17"
+notes = "see https://github.com/freedomofpress/securedrop-engineering/pull/87"
+
+[[trusted.hyper]]
+criteria = "safe-to-deploy"
+user-id = 359 # Sean McArthur (seanmonstar)
+start = "2019-03-01"
+end = "2024-11-17"
+notes = "see https://github.com/freedomofpress/securedrop-engineering/pull/87"
+
 [[trusted.hyper-tls]]
 criteria = "safe-to-deploy"
 user-id = 359 # Sean McArthur (seanmonstar)
 start = "2019-03-19"
 end = "2024-09-12"
 notes = "per https://github.com/freedomofpress/securedrop-engineering/pull/87"
+
+[[trusted.hyper-util]]
+criteria = "safe-to-deploy"
+user-id = 359 # Sean McArthur (seanmonstar)
+start = "2022-01-15"
+end = "2024-11-17"
+notes = "see https://github.com/freedomofpress/securedrop-engineering/pull/87"
 
 [[trusted.indexmap]]
 criteria = "safe-to-deploy"
@@ -189,6 +231,20 @@ criteria = "safe-to-deploy"
 user-id = 163 # Alex Gaynor (alex)
 start = "2023-03-24"
 end = "2024-08-12"
+notes = "Rust Project member"
+
+[[trusted.pin-project]]
+criteria = "safe-to-deploy"
+user-id = 33035 # Taiki Endo (taiki-e)
+start = "2019-03-02"
+end = "2024-11-17"
+notes = "Rust Project member"
+
+[[trusted.pin-project-internal]]
+criteria = "safe-to-deploy"
+user-id = 33035 # Taiki Endo (taiki-e)
+start = "2019-08-11"
+end = "2024-11-17"
 notes = "Rust Project member"
 
 [[trusted.prettyplease]]
@@ -288,6 +344,13 @@ user-id = 6741 # Alice Ryhl (Darksonn)
 start = "2021-01-12"
 end = "2024-09-12"
 notes = "per https://github.com/freedomofpress/securedrop-engineering/pull/87"
+
+[[trusted.tower-layer]]
+criteria = "safe-to-deploy"
+user-id = 10 # Carl Lerche (carllerche)
+start = "2019-04-27"
+end = "2024-11-17"
+notes = "Rust Project member"
 
 [[trusted.wasm-bindgen-futures]]
 criteria = "safe-to-deploy"

--- a/supply-chain/audits.toml
+++ b/supply-chain/audits.toml
@@ -7,6 +7,11 @@ criteria = "safe-to-run"
 version = "0.4.6"
 notes = "only usage of unsafe is in zstd support, via Unshared, which we're not enabling"
 
+[[audits.atomic-waker]]
+who = "Kunal Mehta <legoktm@debian.org>"
+criteria = "safe-to-run"
+delta = "1.1.0 -> 1.1.2"
+
 [[audits.clang-sys]]
 who = "Kunal Mehta <legoktm@debian.org>"
 criteria = "safe-to-run"
@@ -61,10 +66,26 @@ criteria = "safe-to-run"
 version = "1.0.4"
 notes = "code is fine, cannot vouch for correctness though"
 
+[[audits.rustls-pemfile]]
+who = "Kunal Mehta <legoktm@debian.org>"
+criteria = "safe-to-run"
+delta = "1.0.4 -> 2.1.2"
+
+[[audits.rustls-pki-types]]
+who = "Kunal Mehta <legoktm@debian.org>"
+criteria = "safe-to-run"
+version = "1.7.0"
+notes = "Code is reasonable, none of the business logic related to PKI/TLS was verified"
+
 [[audits.slab]]
 who = "Kunal Mehta <legoktm@debian.org>"
 criteria = "safe-to-run"
 delta = "0.4.8 -> 0.4.9"
+
+[[audits.smallvec]]
+who = "Kunal Mehta <legoktm@debian.org>"
+criteria = "safe-to-run"
+delta = "1.11.1 -> 1.13.2"
 
 [[audits.socket2]]
 who = "Kunal Mehta <legoktm@debian.org>"
@@ -80,6 +101,11 @@ delta = "0.1.1 -> 0.1.2"
 who = "Kunal Mehta <legoktm@debian.org>"
 criteria = "safe-to-run"
 delta = "3.9.0 -> 3.10.0"
+
+[[audits.tower-layer]]
+who = "Kunal Mehta <legoktm@debian.org>"
+criteria = "safe-to-run"
+delta = "0.3.1 -> 0.3.2"
 
 [[audits.tracing]]
 who = "Kunal Mehta <legoktm@debian.org>"

--- a/supply-chain/imports.lock
+++ b/supply-chain/imports.lock
@@ -43,6 +43,13 @@ user-id = 33035
 user-login = "taiki-e"
 user-name = "Taiki Endo"
 
+[[publisher.h2]]
+version = "0.4.5"
+when = "2024-05-17"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
 [[publisher.home]]
 version = "0.5.9"
 when = "2023-12-15"
@@ -50,9 +57,44 @@ user-id = 6202
 user-login = "ehuss"
 user-name = "Eric Huss"
 
+[[publisher.http]]
+version = "1.1.0"
+when = "2024-03-04"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
+[[publisher.http-body-util]]
+version = "0.1.1"
+when = "2024-03-11"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
+[[publisher.hyper]]
+version = "1.3.1"
+when = "2024-04-16"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
 [[publisher.hyper-tls]]
 version = "0.5.0"
 when = "2020-12-29"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
+[[publisher.hyper-tls]]
+version = "0.6.0"
+when = "2023-11-27"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
+[[publisher.hyper-util]]
+version = "0.1.3"
+when = "2024-01-31"
 user-id = 359
 user-login = "seanmonstar"
 user-name = "Sean McArthur"
@@ -99,6 +141,20 @@ user-id = 163
 user-login = "alex"
 user-name = "Alex Gaynor"
 
+[[publisher.pin-project]]
+version = "1.1.5"
+when = "2024-03-05"
+user-id = 33035
+user-login = "taiki-e"
+user-name = "Taiki Endo"
+
+[[publisher.pin-project-internal]]
+version = "1.1.5"
+when = "2024-03-05"
+user-id = 33035
+user-login = "taiki-e"
+user-name = "Taiki Endo"
+
 [[publisher.prettyplease]]
 version = "0.2.16"
 when = "2024-01-02"
@@ -116,6 +172,13 @@ user-name = "David Tolnay"
 [[publisher.reqwest]]
 version = "0.11.24"
 when = "2024-01-31"
+user-id = 359
+user-login = "seanmonstar"
+user-name = "Sean McArthur"
+
+[[publisher.reqwest]]
+version = "0.12.4"
+when = "2024-04-19"
 user-id = 359
 user-login = "seanmonstar"
 user-name = "Sean McArthur"
@@ -265,6 +328,17 @@ who = "Alex Crichton <alex@alexcrichton.com>"
 criteria = "safe-to-deploy"
 delta = "0.2.9 -> 1.0.0"
 notes = "Minor changes leading up to the 1.0.0 release and nothing fundamentally new here."
+
+[[audits.bytecode-alliance.audits.http-body]]
+who = "Pat Hickey <phickey@fastly.com>"
+criteria = "safe-to-deploy"
+version = "1.0.0-rc.2"
+
+[[audits.bytecode-alliance.audits.http-body]]
+who = "Alex Crichton <alex@alexcrichton.com>"
+criteria = "safe-to-deploy"
+delta = "1.0.0-rc.2 -> 1.0.0"
+notes = "Only minor changes made for a stable release."
 
 [[audits.bytecode-alliance.audits.idna]]
 who = "Alex Crichton <alex@alexcrichton.com>"
@@ -915,6 +989,12 @@ criteria = "safe-to-run"
 version = "3.4.0"
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
+[[audits.google.audits.tower]]
+who = "ChromeOS"
+criteria = "safe-to-run"
+version = "0.4.13"
+aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
+
 [[audits.google.audits.tower-service]]
 who = "ChromeOS"
 criteria = "safe-to-run"
@@ -980,6 +1060,16 @@ notes = "sourcegraph-based diff did not see the v0.21.6 tag; I retrieved a local
 who = "David Cook <dcook@divviup.org>"
 criteria = "safe-to-run"
 delta = "0.21.6 -> 0.21.7"
+
+[[audits.isrg.audits.base64]]
+who = "Brandon Pitman <bran@bran.land>"
+criteria = "safe-to-run"
+delta = "0.21.7 -> 0.22.0"
+
+[[audits.isrg.audits.base64]]
+who = "David Cook <dcook@divviup.org>"
+criteria = "safe-to-run"
+delta = "0.22.0 -> 0.22.1"
 
 [[audits.isrg.audits.once_cell]]
 who = "Brandon Pitman <bran@bran.land>"
@@ -1209,6 +1299,12 @@ There are also several more places where DWARF data is mmapped from a filesystem
 then loaded. These appear to all derive from existing paths that themselves were already
 being mmapped and loaded.
 """
+aggregated-from = "https://raw.githubusercontent.com/zcash/zcash/master/qa/supply-chain/audits.toml"
+
+[[audits.zcash.audits.base64]]
+who = "Daira-Emma Hopwood <daira@jacaranda.org>"
+criteria = "safe-to-deploy"
+delta = "0.21.5 -> 0.21.7"
 aggregated-from = "https://raw.githubusercontent.com/zcash/zcash/master/qa/supply-chain/audits.toml"
 
 [[audits.zcash.audits.bytes]]

--- a/supply-chain/imports.lock
+++ b/supply-chain/imports.lock
@@ -1,6 +1,13 @@
 
 # cargo-vet imports lock
 
+[[publisher.atomic-waker]]
+version = "1.1.0"
+when = "2023-01-15"
+user-id = 33035
+user-login = "taiki-e"
+user-name = "Taiki Endo"
+
 [[publisher.cexpr]]
 version = "0.6.0"
 when = "2021-10-11"
@@ -74,13 +81,6 @@ user-name = "Sean McArthur"
 [[publisher.hyper]]
 version = "1.3.1"
 when = "2024-04-16"
-user-id = 359
-user-login = "seanmonstar"
-user-name = "Sean McArthur"
-
-[[publisher.hyper-tls]]
-version = "0.5.0"
-when = "2020-12-29"
 user-id = 359
 user-login = "seanmonstar"
 user-name = "Sean McArthur"
@@ -168,13 +168,6 @@ when = "2024-01-21"
 user-id = 3618
 user-login = "dtolnay"
 user-name = "David Tolnay"
-
-[[publisher.reqwest]]
-version = "0.11.24"
-when = "2024-01-31"
-user-id = 359
-user-login = "seanmonstar"
-user-name = "Sean McArthur"
 
 [[publisher.reqwest]]
 version = "0.12.4"
@@ -322,12 +315,6 @@ notes = """
 Still looks like a good DWARF-parsing crate, nothing major was added or deleted
 and no `unsafe` code to review here.
 """
-
-[[audits.bytecode-alliance.audits.http]]
-who = "Alex Crichton <alex@alexcrichton.com>"
-criteria = "safe-to-deploy"
-delta = "0.2.9 -> 1.0.0"
-notes = "Minor changes leading up to the 1.0.0 release and nothing fundamentally new here."
 
 [[audits.bytecode-alliance.audits.http-body]]
 who = "Pat Hickey <phickey@fastly.com>"
@@ -656,42 +643,6 @@ criteria = "safe-to-deploy"
 version = "0.3.1"
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
-[[audits.google.audits.h2]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-version = "0.3.18"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.h2]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-delta = "0.3.18 -> 0.3.19"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.h2]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-delta = "0.3.19 -> 0.3.20"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.h2]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-delta = "0.3.20 -> 0.3.21"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.h2]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-delta = "0.3.21 -> 0.3.24"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.h2]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-delta = "0.3.24 -> 0.3.26"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
 [[audits.google.audits.hashbrown]]
 who = "Nicholas Bishop <nicholasbishop@google.com>"
 criteria = "safe-to-run"
@@ -704,34 +655,10 @@ criteria = "safe-to-run"
 delta = "0.13.2 -> 0.14.3"
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
-[[audits.google.audits.http]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-version = "0.2.9"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.http-body]]
-who = "ChromeOS"
-criteria = "safe-to-run"
-version = "0.4.5"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
 [[audits.google.audits.httparse]]
 who = "George Burgess IV <gbiv@google.com>"
 criteria = "safe-to-run"
 version = "1.8.0"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.httpdate]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-deploy"
-version = "1.0.3"
-aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
-
-[[audits.google.audits.hyper]]
-who = "George Burgess IV <gbiv@google.com>"
-criteria = "safe-to-run"
-version = "0.14.27"
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
 [[audits.google.audits.itertools]]
@@ -964,6 +891,12 @@ criteria = "safe-to-run"
 version = "0.4.7"
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
+[[audits.google.audits.smallvec]]
+who = "Android Legacy"
+criteria = "safe-to-run"
+version = "1.6.1"
+aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
+
 [[audits.google.audits.socket2]]
 who = "Vovo Yang <vovoy@google.com>"
 criteria = "safe-to-run"
@@ -993,6 +926,12 @@ aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust
 who = "ChromeOS"
 criteria = "safe-to-run"
 version = "0.4.13"
+aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
+
+[[audits.google.audits.tower-layer]]
+who = "ChromeOS"
+criteria = "safe-to-run"
+version = "0.3.1"
 aggregated-from = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/refs/heads/main/cargo-vet/audits.toml?format=TEXT"
 
 [[audits.google.audits.tower-service]]
@@ -1049,17 +988,6 @@ delta = "0.21.3 -> 0.21.4"
 who = "David Cook <dcook@divviup.org>"
 criteria = "safe-to-run"
 delta = "0.21.4 -> 0.21.5"
-
-[[audits.isrg.audits.base64]]
-who = "Brandon Pitman <bran@bran.land>"
-criteria = "safe-to-run"
-delta = "0.21.5 -> 0.21.6"
-notes = "sourcegraph-based diff did not see the v0.21.6 tag; I retrieved a local copy of the repo and used that for diff'ing."
-
-[[audits.isrg.audits.base64]]
-who = "David Cook <dcook@divviup.org>"
-criteria = "safe-to-run"
-delta = "0.21.6 -> 0.21.7"
 
 [[audits.isrg.audits.base64]]
 who = "Brandon Pitman <bran@bran.land>"
@@ -1273,6 +1201,12 @@ delta = "0.3.0 -> 0.4.0"
 notes = "Primarily adding a no_std mode"
 aggregated-from = "https://raw.githubusercontent.com/freedomofpress/securedrop/develop/supply-chain/audits.toml"
 
+[[audits.securedrop.audits.smallvec]]
+who = "Kunal Mehta <legoktm@debian.org>"
+criteria = "safe-to-run"
+delta = "1.6.1 -> 1.11.1"
+aggregated-from = "https://raw.githubusercontent.com/freedomofpress/securedrop/develop/supply-chain/audits.toml"
+
 [[audits.zcash.audits.anyhow]]
 who = "Jack Grigg <jack@electriccoin.co>"
 criteria = "safe-to-deploy"
@@ -1387,24 +1321,6 @@ notes = """
   correct; the slice's lifetime logically comes from the `AsyncBufRead` reader
   inside `FillBuf`, rather than the `Context`.
 """
-aggregated-from = "https://raw.githubusercontent.com/zcash/zcash/master/qa/supply-chain/audits.toml"
-
-[[audits.zcash.audits.http]]
-who = "Jack Grigg <jack@electriccoin.co>"
-criteria = "safe-to-deploy"
-delta = "1.0.0 -> 0.2.11"
-aggregated-from = "https://raw.githubusercontent.com/zcash/zcash/master/qa/supply-chain/audits.toml"
-
-[[audits.zcash.audits.http-body]]
-who = "Jack Grigg <jack@electriccoin.co>"
-criteria = "safe-to-deploy"
-delta = "0.4.5 -> 0.4.6"
-aggregated-from = "https://raw.githubusercontent.com/zcash/zcash/master/qa/supply-chain/audits.toml"
-
-[[audits.zcash.audits.hyper]]
-who = "Jack Grigg <jack@electriccoin.co>"
-criteria = "safe-to-deploy"
-delta = "0.14.27 -> 0.14.28"
 aggregated-from = "https://raw.githubusercontent.com/zcash/zcash/master/qa/supply-chain/audits.toml"
 
 [[audits.zcash.audits.ipnet]]

